cg/cli/delete/observations.py

```diff
@@ -0,0 +1,35 @@
+import logging
+
+import click
+from cg.apps.loqus import LoqusdbAPI
+from cg.models.cg_config import CGConfig
+from cg.store import Store, models
+
+LOG = logging.getLogger(__name__)
+
+
+@click.group("delete")
+def delete_cmd():
+    """Delete variants associated to a case in LoqusDB."""
+    pass
+
+
+@delete_cmd.command()
+@click.option("-c", "--case_id", help="internal case id")
+@click.pass_obj
```

Comment:
I would feel more comfortable if there also was a dry-run option that would do all except deleting. Removing stuff is hazardous processes. There is global click dry-run constant in: cg/constants/constants.py for import and reuse